public/js/app.Ctrl.js

```diff
@@ -0,0 +1,201 @@
+App.controller('App.Ctrl', function($scope, $timeout, AppService) {
```

Comment:
Definitely use the array method of including services in angular, if this code gets minified/compressed, it won't work.
someModule.controller('MyController', ['$scope', 'dep1', 'dep2', function($scope, dep1, dep2) {....
Do this for all of angular, including services, etc.